### PR TITLE
12.0 mig partner name vat

### PR DESCRIPTION
**Pull request by VrajaTechnologies**

### CODE REVIEW

File: partner_name_vat/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Eric Antones <eantones@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Partner name with vat",
+    "summary": "This module concatenates vat number to partner name on display_name",
+    "version": "12.0.02.03.21",
```
Comment (eantones) [inline partner_name_vat/__manifest__.py]: This should be 12.0.1.0.0 on the first commit of the migrations as it's described here https://github.com/nuobit/odoo-addons/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration.
@VrajaTechnologies can you please change it and push a new commit

File: partner_name_vat/models/models.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    @api.multi
+    # @api.multi
```
Comment (eantones) [inline partner_name_vat/models/models.py]: Remove this line

**Review by eantones — CHANGES_REQUESTED** [review]
I'll merge the PR once this two minor things will be fixed.Thanks

**Comment by eantones** [comment]
@VrajaTechnologies Travis is red, it seems you forgot to execute pre-commit, please fix it and push it again. Thanks

**Comment by eantones** [comment]
@VrajaTechnologies Please, fix Travis errors and squash the "Migration to 12.0" commits into one when you push it. It should only be 3 commits in total. Thanks

**Comment by VrajaTechnologies** [comment]
Hi Eric,

We have checked in Git Hub   those 3 modules successfully build.

I have checked *partner_default_journal, *This module does not build in git
hub. *partner_default_journal *Once I have pre commit this module it was
successfully pass all cases nothing to fail in that process.

Kindly check provide us some information because I have checked different
different cases but not succeed.


Thanks In Advance

Best Regards,

Vraja Technologies

Vraja Technologies
*Odoo ERP Consultancy Service Provider*
207, Time Square, Ayodhya Chowk, • 150 feet ring road, Rajkot - 360006
*India*.
*Skype* :  *live:vrajatechnologies*
[image: email-icon]  ***@***.***
***@***.***>*
[image: website-icon]  *www.vrajatechnologies.com
<https://www.vrajatechnologies.com/>*

**Comment by eantones** [comment]
Hi @VrajaTechnologies Travis is red, please check the errors here https://travis-ci.com/github/nuobit/odoo-addons/jobs/500520718. It seems you forgot to execute pre-commit before pushing the commit. Please fix it and push it again but, in order to avoid having duplicated "Migration" commits, also squash with the previous "Migration to 12.0" commit before doing it. The migration commit should only be one, otherwise it's difficult to see clearly the changes made. Thanks

**Pull request merged**